portfolio/src/main/webapp/index.html

```diff
@@ -1,77 +1,90 @@
+		</header>
+		<div id="content">
+			<section id="home">
+				<div id="home-text">
+					<h2> Welcome. <br /> My name is Denzil Bilson, and I am a budding Software Engineer.</h2>
+				</div>
+				<img src="/images/denzil.jpg" id="denzilpicture" alt="picture of Denzil Bilson">
+			</section>
+			<section id="about-me">
+				<div id="experiences">
+					<div class="block">
+						<div class="box">
+							<img src="https://upload.wikimedia.org/wikipedia/commons/6/61/HTML5_logo_and_wordmark.svg" alt="picture of HTML5 logo">
+						</div>
+						<div class="box">
+							<img src="https://upload.wikimedia.org/wikipedia/commons/3/3d/CSS.3.svg" alt="picture of CSS3 logo">
+						</div>
+						<div class="box">
+							<img src="https://upload.wikimedia.org/wikipedia/commons/b/ba/Javascript_badge.svg" alt="picture of Javascript logo">
+						</div>
+					</div>
+					<div class="block">
+						<div class="box">
+							<img src="https://upload.wikimedia.org/wikipedia/commons/1/18/ISO_C%2B%2B_Logo.svg" alt="picture of C++ logo">
+						</div>
+						<div class="box">
+							<img src="https://upload.wikimedia.org/wikipedia/commons/c/c3/Python-logo-notext.svg" alt="picture of Python logo">
+						</div>
+						<div class="box">
+							<img src="https://upload.wikimedia.org/wikipedia/commons/5/53/Google_%22G%22_Logo.svg" alt="picture of Google logo">                        </div>
+					</div>
+				</div>
+				<div id="about-me-text">
+					<h2>Who am I?</h2>
+					<p>No project is hard enough. No venture too far-fetched. I came into the field of tech only a couple
+					years ago, but my passion for technology and self-excellence have brought me far. With projects spanning
+					from simple games to entire programs based on data structure utilizing organizational algorithms, I have barely scratched the
+					surface of what is possible with technology and software. Want to find out what is?
+					</p>
+				</div>
+			</section>
+			<section id="projects">
+				<div class="block">
+					<a href="http://project-conduit.appspot.com/" target="_blank"><img src="/images/pong.png" alt="picture of pong game"></a>
+					<a href="http://caffeinecap.appspot.com/" target="_blank"><img src="/images/caffeinecap.png" alt="picture of caffeine cap website"></a>
+					<a href="https://www.youtube.com/watch?v=Dt9rFrC3FhQ" target="_blank"><img src="/images/pong_youtube.png" alt="picture of pong video on youtube"></a>
+				</div>
+		</div>
+			</section>
+			<footer>
+				<ul>
+					<li><a target="_blank" class="icon" href="https://www.linkedin.com/in/denzilbilson"><i class="fa fa-fw fa-2x fa-inverse fa-linkedin"></i></a></li>
+					<li><a target="_blank" class="icon" href="https://www.github.com/denzilbilson-google"><i class="fa fa-fw fa-2x fa-inverse fa-github"></i></a></li>
+					<li><a target="_blank" class="icon" href="mailto:denzilbilson@google.com"><i class="fa fa-fw fa-2x fa-inverse fa-envelope"></i></a></li>
+				</ul>
+			</footer>
+			</section>
+			<section>
```

Comment:
Just double checking: did you mean to put the comments below the footer? I think of footers as being the last thing on a page, but it's a design choice that's completely up to you.